crates/torii/server/src/handlers/mcp.rs

```diff
@@ -0,0 +1,542 @@
+use std::sync::Arc;
+
+use base64::engine::general_purpose::STANDARD;
+use base64::Engine;
+use futures_util::{SinkExt, StreamExt};
+use hyper::{Body, Request, Response, StatusCode};
+use serde::{Deserialize, Serialize};
+use serde_json::{json, Value};
+use sqlx::{Column, Row, SqlitePool, TypeInfo};
+use tokio_tungstenite::tungstenite::Message;
+
+use super::Handler;
+
+const JSONRPC_VERSION: &str = "2.0";
+const MCP_VERSION: &str = "2024-11-05";
```

Comment:
Do you have a link to the MCP releases to know how to bump this in case it becomes necessary?

Reply:
Yes, it uses the latest date of the release. I took it from here: https://github.com/modelcontextprotocol/typescript-sdk/blob/main/src/types.ts#L3